crates/mun_hir/src/code_model/function.rs

```diff
@@ -103,6 +103,21 @@ impl FunctionData {
     pub fn is_extern(&self) -> bool {
         self.flags.is_extern()
     }
+
+    /// Returns true if the first param is `self`. This is relevant to decided
+    /// whether this can be called as method as opposed to an associated
```

Comment:
```suggestion
    /// Returns true if the first param is `self`. This is relevant to decide
    /// whether this can be called as a method as opposed to an associated
```